Gemfile

```diff
@@ -0,0 +1,22 @@
+source 'https://rubygems.org'
+
+# Ruby version
+ruby '3.2.1'
```

Comment:
[suggestion]
Bump this version to the more recent version 3.3, it is probably better to not specify that patch version.